### ISSUE TITLE
Improve z axis micromanipulator stability

### ISSUE THREAD

**Issue by amchagas** [amchagas/Flypi#40]
The current version of the z axis of the micromanipulator consists of an M4 screw and a single locked nut that act as the axis carrier. this makes the axis a bit wobbly. 

A possible solution for this would be to add a second nut at a certain distance from the previous one similar to what is present on the other axis. 

Other improvements to be implemented are:
- Adding a spring (or two) to the axis to decrease backlash
- improving the pockets where the nuts go in.

Here are the files for the manipulator: 

Thumb Screws: https://github.com/amchagas/Flypi/blob/master/3D_print_files/v2.0/STEPs/Manipulator_thumbscrew.step

Base: https://github.com/amchagas/Flypi/blob/master/3D_print_files/v2.0/STEPs/ManipulatorBase.stp

X axis: https://github.com/amchagas/Flypi/blob/master/3D_print_files/v2.0/STEPs/Manipulator_Xaxis.step

Y axis: https://github.com/amchagas/Flypi/blob/master/3D_print_files/v2.0/STEPs/manipulator_Yaxis.step

z axis: https://github.com/amchagas/Flypi/blob/master/3D_print_files/v2.0/STEPs/Manipulator_Zaxis.step

**Comment by isobianin** [comment]
Hello. So I have made some progress on this particular issue (#40) and the next one (#41) and submitted proposal to this job on upwork. Here I will dublicate screenshots of design. Can you please tell me whether I need to continue working on this project?
![2_holder_front](https://user-images.githubusercontent.com/52375129/60506642-f2126a80-9cce-11e9-8a9f-f546d3356434.jpg)
![3_interchangeable system](https://user-images.githubusercontent.com/52375129/60506644-f2126a80-9cce-11e9-8ef3-625e1f2e2f0d.jpg)
![1_holder](https://user-images.githubusercontent.com/52375129/60506645-f2126a80-9cce-11e9-951a-802850089911.jpg)

**Comment by amchagas** [comment]
Hi @ODESSARAMBO this looks very nice! 


Thanks for the effort in putting it together. 


As you probably saw, that Upwork offer is closed already, since we learned after putting it online that, because of the regulations of the funding agency we received a grant from,  we can only work with freelancers that can emit a bill with a VAT number. 
We created a new job post stating this fact https://www.upwork.com/ab/applicants/1145797064325591040/job-details

I'm sorry about this confusion.   

If you are able to provide us with a bill that has VAT and VAT number, then we can keep discussing things!

best,
Andre
